src/couch_server.erl

```diff
@@ -549,9 +554,13 @@ delete_db_file(RootDir, FullFilePath, Options) ->
     end.
 
 rename_on_delete(Original) ->
+    file:rename(Original, deleted_filename(Original)).
+
+deleted_filename(Original) ->
     {{Y,Mon,D}, {H,Min,S}} = calendar:universal_time(),
+    UUID = ?b2l(couch_uuids:random()),
```

Comment:
Good idea. I'll change it to return:
- `{ok, deleted}`
- `{ok, {renamed, FileName}}`
- `{error, Reason}`